Dfe.PrepareConversions/Dfe.PrepareConversions.Data/Features/ApiClient.cs

```diff
@@ -55,4 +56,13 @@ public async Task<HttpResponseMessage> AddProjectNote(int id, AddProjectNote pro
    {
       return await AcademisationClient.PostAsync(string.Format(PathFor.AddProjectNote, id), JsonContent.Create(projectNote));
    }
+
+   public async Task<HttpResponseMessage> SetProjectExternalApplicationForm(int id, bool externalApplicationFormSaved, string externalApplicationFormUrl)
+   {
+      dynamic payload = new ExpandoObject();
```

Comment:
Looks good, more for my learning than anything I have a question, why use this over a static type? Would that not give more clarity to the contract - or do we foresee this changing or is it something to do with how you expect the payload to be manipulated?